tests/cypress/integration/forms_spec.js

```diff
@@ -0,0 +1,254 @@
+/// <reference types="cypress" />
+import {
+  standardFormUrls,
+  interactiveForms,
+  formsWithEmailTestId,
+} from "../utils";
+
+context("Marketo forms", () => {
+  beforeEach(() => {
+    cy.intercept({
+      method: "POST",
+      url: "/marketo/submit",
+    }).as("captureLead");
+  });
+
+  afterEach(() => {
+    cy.wait("@captureLead").then(({ request, response }) => {
+      expect(request.method).to.equal("POST");
+      expect(response.statusCode).to.equal(302);
+    });
+  });
+
+  // test static forms on /contact-us page
+  it("should check each contact form on /contact-us pages with standard form", () => {
+    cy.visit("/");
+    cy.acceptCookiePolicy();
+    standardFormUrls.forEach((url) => {
+      cy.visit(url);
+      cy.findByLabelText(/First name:/).type("Test");
+      cy.findByLabelText(/Last name:/).type("Test");
+      cy.findByLabelText(/Email address:/).type("test@test.com");
+      cy.findByLabelText(/Mobile\/cell phone number:/).type("07777777777");
+      cy.findByLabelText(/Country:/).select("Colombia");
+      cy.findByLabelText(/Company name:/).type("Test");
+      cy.findByLabelText(/Job title:/).type("test", {
+        force: true,
+      });
+      cy.findByLabelText(/What would you like to talk to us about?/).type(
+        "test test test test"
+      );
+      cy.findByLabelText(/I agree to receive information/).click({
+        force: true,
+      });
+      cy.findByText(/Submit/).click({
+        force: true,
+      });
+      cy.findAllByText(/Thank you/).should("be.visible");
```

Comment:
This is fine, but can lead to false positives in the future. Any chance we could make this more specific? Some ideas:
- Do we always go to a specific URL on a success? Could we check for that URL?
- Is "Thank you" in a heading element - in this case we should use `findByRole("heading", { name: /Thank you/ })`

Reply:
Updated to `cy.url().should("include", "#success");` and `cy.findByRole("heading", { name: /Thank you/ });`